udata/templates/territories/search-result.html

```diff
@@ -1,11 +1,11 @@
 {% cache cache_duration, 'territory-search-result', territory.id, g.lang_code %}
-{% set logo=territory.logo_url() or theme_static('img/placeholder_territory.png') %}
+{% set logo=territory.logo_url() or theme_static('img/placeholder_territory_medium.png') %}
 <li class="search-result territory-result">
     <a href="{{ territory.url }}" title="{{ territory.name }}">
         <div class="result-logo pull-left">
             <img alt="{{ territory.name }}"
                 src="{{ logo }}"
-                width="70" height="70">
+                width="64" height="70">
```

Comment:
All logos have different sizes and the general case is the placeholder which matches blazons so I think it's a good trade off.